content/en/news/releases/1.12.x/announcing-1.12.2/index.md

```diff
@@ -64,5 +64,5 @@ Introduced a flag `meshConfig.defaultConfig.proxyMetadata.INVALID_DROP` to contr
 - **Fixed** changes in delegate virtual service not taking effect when RDS cache enabled.
   ([Issue #36525](https://github.com/istio/istio/issues/36525))
 
-- **Fixed** an issue when using Envoy [`v3alpha`](https://www.envoyproxy.io/docs/envoy/latest/version_history/v1.20.0#incompatible-behavior-changes) APIs in `EnvoyFilter`s.
+- **Fixed** an issue when using Envoy [`v3alpha`](https://www.envoyproxy.io/docs/envoy/latest/version_history/v1.20.0#incompatible-behavior-changes) APIs in `EnvoyFilter`.
```

Comment:
Yes, I quite agree with you. I have a small suggestion. Shall we unify this specification for the unification of documents

Reply:
My comment above is I prefer to keep it as is with the *s* outside the backticks.

Reply:
Ok, I will close this issue. I would like to ask you something. How can I become a document reviewer，boss

Reply:
https://github.com/istio/community/blob/master/ROLES.md#documentation-reviewer is the official requirements. Since there really is no longer a Doc WG, I suspect @craigbox or I would be the WG Lead although there are other maintainers (just not as active).

Reply:
I have read the official requirements, and I basically meet the requirements, so what should I do? The following